core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1502,6 +1505,30 @@ public ServerBuilder contentPreview(int length) {
         return contentPreview(length, ArmeriaHttpUtil.HTTP_DEFAULT_CONTENT_CHARSET);
     }
 
+    /**
+     * Sets whether the response header will include Server header.
+     */
+    public ServerBuilder setDefaultServerNameResponseHeader(boolean defaultServerNameResponseHeader) {
```

Comment:
We don't use `set` prefix throughout this class. How about just `includeServerHeader`? Not sure this is the best name. /cc @trustin and @ikhoon

Reply:
Fixed to use `useServerHeader`.